django_sorcery/db/strategy_options.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, unicode_literals
+
+from sqlalchemy.orm.interfaces import MapperOption
+
+from django.core.exceptions import ImproperlyConfigured
+
+
+def _key_from_query(query):
+    """
+    Given a Query, create a cache key.
+
+    There are many approaches to this; here we use the simplest, which is to create an md5 hash of the text of the SQL
+    statement, combined with stringified versions of all the bound parameters within it. There's a bit of a
+    performance hit with compiling out "query.statement" here; other approaches include setting up an explicit cache
+    key with a particular Query, then combining that with the bound parameter values.
+    """
+
+    stmt = query.with_labels().statement
+    compiled = stmt.compile()
+    params = compiled.params
```

Comment:
in [explain](https://github.com/miki725/django-debug-toolbar-alchemy/blob/master/debug_toolbar_alchemy/panels/sql/tracking.py#L84-L108) had to do some magic to account for multiparams, etc. maybe do something similar here because I suspect those more intricate queries will be more critical to get cache key correctly...

Reply:
i don't think multiparams is supported for select statements by any db